www/docs/server/adapters-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+id: adapters-intro
+title: Adapters
+sidebar_label: Adapters
+slug: /server/adapters
+---
+
+tRPC is not a backend on its own, and must therefore be served using other hosts, such as a simple [Node.js HTTP Server](adapters/standalone), [Express](adapters/express), or even [Next.js](adapters/nextjs). Most tRPC features are the same no matter which backend you choose. **Adapters** act as the glue between the host system and your tRPC API.
+
+Adapters typically follow some common conventions, allowing you to set up context creation via `createContext`, and globally handle errors via `onError`, but importantly allow you to choose an appropriate host for your application.
+
+We support many modes of hosting an API, which you will find documented here.
+
+- For serverful APIs, you might want our [Standalone](adapters/standalone) adapter, or use the [Express](adapters/express) or [Fastify](adapters/fastify) adapters to hook into your existing APIs.
+- You might want a serverless solution and choose [AWS Lambda](adapters/aws-lambda) or [Fetch](adapters/fetch). You might have a full-stack framework and want a full integration like [Next.js](adapters/nextjs), or you could use the [Fetch](adapters/fetch) adapter with Next.js, Astro, Remix, or SolidStart.
+- For edge runtimes, refer to the [Fetch](adapters/fetch) adapter that builds on Web APIs.
```

Comment:
```suggestion
- For serverful APIs, you might want our [Standalone](adapters/standalone) adapter, or use the [Express](adapters/express) or [Fastify](adapters/fastify) adapters to hook into your existing APIs
- You might want a serverless solution and choose [AWS Lambda](adapters/aws-lambda) or [Fetch](adapters/fetch). You might have a full-stack framework and want a full integration like [Next.js](adapters/nextjs), or you could use the [Fetch](adapters/fetch) adapter with Next.js, Astro, Remix, or SolidStart
- For edge runtimes, refer to the [Fetch](adapters/fetch) adapter that builds on Web APIs.
```

I've been taught that lists should have a `.` on the final item only, good shout to turn it into a list though :)